TFNetworkRecLayer.py

```diff
@@ -4428,6 +4430,23 @@ def transform_config_dict(cls, d, network, get_layer):
         [get_layer(name) for name in d["explicit_search_sources"]] if network.search_flag else [])
     super(ChoiceLayer, cls).transform_config_dict(d, network=network, get_layer=get_layer)
 
+  @classmethod
+  def _create_search_beam(cls, name, beam_size, sources, network):
+    """
+    :param str name:
+    :param int beam_size:
+    :param list[LayerBase] sources:
+    :param TFNetwork.TFNetwork network:
+    :rtype: SearchBeam
```

Comment:
I think the type is wrong, if it is not globally imported. (Did you check the type inference for the function?) I guess it must be `TFUtil.SearchBeam`.

Reply:
True, fixed now.